packages/collector/test/tracing/cloud/aws/s3/app.js

```diff
@@ -188,7 +195,7 @@ availableOperations.forEach(operation => {
         const data = await S3Api.runOperation(operation, method, withError);
         res.send(data);
       } catch (err) {
-        res.status(500).send({
+        res.status(200).send({
```

Comment:
I wonder why we need to mask an error like this since the test already uses `appControls.sendRequest` with `simple: withError === false`. Shouldn't that take care of the HTTP 500 on the client side?

Reply:
As per our talk on Slack, you are right. The real problem was the lack of handling for undefined object when attempting to log `options.Bucket`.